MetaMask/metamask-extension
# UX: Multichain: Focus on account search when menu appears

## Explanation

When the multichain account menu pops up, focus stays in the background, which is bad for accessibility.  With this change, the account menu's search box will be focused for quick searching and ability to tab to account items.  It's important to note that `autoFocus` is not working for `TextFieldSearch` so we need to go this route instead.

## Manual Testing Steps

1.  Click the account in the header to open the menu
2. See the INPUT element receives focus
3. Press TAB a few times to navigate around the account list

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
